presentation/src/main/java/com/example/user/test/presentation/screen/characters/CharactersActivity.kt

```diff
@@ -2,28 +2,32 @@ package com.example.user.test.presentation.screen.characters
 
 import android.content.Context
 import android.content.Intent
-import android.os.Bundle
 import android.support.v4.app.Fragment
 import com.arellomobile.mvp.presenter.InjectPresenter
 import com.arellomobile.mvp.presenter.ProvidePresenter
 import com.example.user.test.R
 import com.example.user.test.presentation.app.MarvelApplication
 import com.example.user.test.presentation.base.BaseActivity
-import com.example.user.test.presentation.base.BaseNavigator
+import com.example.user.test.presentation.base.FlowNavigator
 import com.example.user.test.presentation.base.FlowRouter
 import com.example.user.test.presentation.screen.Screens
 import com.example.user.test.presentation.screen.characters.characters.CharacterFragment
 import com.example.user.test.presentation.screen.characters.favorite.FavoriteFragment
 import com.example.user.test.presentation.screen.characters.profile.ProfileFragment
 import ru.terrakok.cicerone.Navigator
+import javax.inject.Inject
 
 class CharactersActivity : BaseActivity<CharactersPresenter, FlowRouter>(), CharactersView {
 
     companion object {
         fun getInstance(mCtx: Context): Intent = Intent(mCtx, CharactersActivity::class.java)
     }
 
-    override val navigator: Navigator = object : BaseNavigator(this, R.id.tlFragmentContainer) {
+    init {
```

Comment:
You must inject according to `Fragment/Activity` lifecycle. Move this logic to `onCreate`.